scalyr_agent/agent_main.py

```diff
@@ -213,7 +213,7 @@ def main(self, config_file_path, command, command_options):
         self.__config_file_path = config_file_path
 
         try:
-            self.__config = self.__read_and_verify_config(config_file_path)
+            self.__config = self.__read_and_verify_config(config_file_path, force_http=command_options.force_http )
```

Comment:
Ok, sorry to ask for a variable rename, but let's call this `allow_http` instead of `force_http`.  It should be a configuration option that we can have in agent.json.

Reply:
No problem.